### PR TITLE
MOTECH-2558 fixed disabled submit button

### PR DESCRIPTION
**Pull request by jablonskidawid**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 1]
```diff
@@ -1,3 +1,15 @@
+var bootstrapApp = angular.module('bootstrapApp',[]);
+bootstrapApp.controller('controller', function($scope) {
+              $scope.insertValuesToScopes = function() {
```
Comment (pgesek) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: why aren't we using ng-model for this?

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 2]
```diff
@@ -1,3 +1,15 @@
+angular.module('bootstrapApp',[])
+.controller('controller', ['$scope', function($scope) {
+              $scope.insertValuesToScopes = function() {
```
Comment (pgesek) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: why aren't we using ng-model for this?

File: platform/server/src/main/webapp/WEB-INF/jsp/bootstrapconfig.jsp [hunk 3]
```diff
@@ -23,7 +21,7 @@
         </script>
     </c:if>
 </head>
-<body class="body-startup" ng-app="bootstrapApp" >
+<body id="mainBody" class="body-startup" ng-app="bootstrapApp" ng-controller="controller" >
```
Comment (martokarski) [inline platform/server/src/main/webapp/WEB-INF/jsp/bootstrapconfig.jsp]: body needs an id?

Comment (martokarski) [inline platform/server/src/main/webapp/WEB-INF/jsp/bootstrapconfig.jsp]: Controller name is too general. Maybe better will be 'formController' or 'bootstrapFormController'.

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 4]
```diff
@@ -1,29 +1,11 @@
...
-				       setSuggestedValueByName(bootstrapObj[key]["name"], bootstrapObj[key]["value"]);
-				}
-				sessionStorage.removeItem("bootstrapString");
-			 }
-		});
-
-})();
-
+angular.module('bootstrapApp',[])
+.controller('controller', ['$scope', function($scope, $sessionStorage) {
```
Comment (martokarski) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: '$sessionStorage' is never used

**Comment by martokarski** [comment]
I cannot save the config settings. It always returns an error and sometimes redirects to error page "Error 500: Internal Server Error".

**Comment by martokarski** [comment]
Looks good to me.

**Comment by kmadej** [comment]
test this please

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 5]
```diff
@@ -1,29 +1,13 @@
...
-				}
-				sessionStorage.removeItem("bootstrapString");
-			 }
-		});
-
-})();
+angular.module('bootstrapApp',[])
+.controller('bootstrapFormController', ['$scope', function($scope) {
+    if (sessionStorage.config != null){
+    $scope.config = angular.fromJson(sessionStorage.config);
```
Comment (kmadej) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: please add a tabulation here

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 6]
```diff
@@ -1,29 +1,13 @@
...
-				       setSuggestedValueByName(bootstrapObj[key]["name"], bootstrapObj[key]["value"]);
-				}
-				sessionStorage.removeItem("bootstrapString");
-			 }
-		});
-
-})();
+angular.module('bootstrapApp',[])
+.controller('bootstrapFormController', ['$scope', function($scope) {
+    if (sessionStorage.config != null){
```
Comment (kmadej) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: add space after )

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 7]
```diff
@@ -1,29 +1,13 @@
...
+    $scope.config = angular.fromJson(sessionStorage.config);
+    }
 
+    $scope.saveBootstrapData = function(){
```
Comment (kmadej) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: add space after ()

File: platform/server/src/main/webapp/static/js/bootstrap-page.js [hunk 8]
```diff
@@ -1,29 +1,13 @@
...
+     sessionStorage.config = angular.toJson($scope.config);
```
Comment (kmadej) [inline platform/server/src/main/webapp/static/js/bootstrap-page.js]: please add a tabulation here

**Comment by kmadej** [comment]
test this please

**Comment by kmadej** [comment]
test this please

**Pull request merged**